src/v2/Apps/Partner/Components/PartnerViewingRooms/ViewingRoomCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { Image, ResponsiveBox, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ViewingRoomCard_viewingRoom } from "v2/__generated__/ViewingRoomCard_viewingRoom.graphql"
+import { cropped } from "v2/Utils/resized"
+
+interface ViewingRoomCardProps {
+  viewingRoom: ViewingRoomCard_viewingRoom
+}
+
+const ViewingRoomCard: React.FC<ViewingRoomCardProps> = ({ viewingRoom }) => {
+  const { coverImage, title, href, exhibitionPeriod } = viewingRoom
+
+  const coverImageURL = cropped(coverImage?.imageURLs?.normalized ?? "", {
+    height: 222,
+    width: 263,
+  })
+
+  return (
+    <RouterLink to={href} textDecoration="none">
+      {coverImage && (
+        <ResponsiveBox aspectWidth={263} aspectHeight={222} maxWidth="100%">
+          <Image
+            {...coverImageURL}
+            alt={title}
+            width="100%"
+            height="100%"
+            lazyLoad
+          />
+        </ResponsiveBox>
+      )}
+      {coverImage && title && href && exhibitionPeriod && (
```

Comment:
This condition reads a bit funny to me - why we only display `Viewing Room` only if all of those are present? What will happen if let's say we are missing only one?

Should we either display it always or not display it ever?

Reply:
I added this condition for VR label because of this test:

https://github.com/artsy/force/blob/bae52d423f07ea21f7dbe11155cdd5f0671b683f/src/v2/Apps/Partner/Components/__tests__/PartnerViewingRooms/ViewingRoomCard.jest.tsx#L65